lib/include/event-driven/vDraw.h

```diff
@@ -1228,100 +181,15 @@ class skinDraw : public vDraw {
 
 class skinsampleDraw : public vDraw {
 
-
-private:
-
-    loadMap tmap;
-    cv::Mat baseimage;
-    int radius,noise,max_value;
-
 public:
-  void initialise(){
 
-        Xlimit = 800;
-        Ylimit = 800;
-
-        baseimage = cv::Mat(cv::Size(Xlimit, Ylimit), CV_8UC3);
-        baseimage.setTo(255);
-
-        tmap.initialise("leftHand");
-
-        radius = tmap.radius;
-        noise = tmap.noise;
-        max_value=tmap.max_value;
-
-        int x;
-        int y;
-
-        
-        for (auto it = tmap.pos.begin(); it != tmap.pos.end(); it++)
-        {
-            x = tmap.xoffset + tmap.scaling* std :: get<0>(it->second);
-            y =  tmap.yoffset +tmap.scaling* std :: get<1>(it->second);
-            cv::Point centr_all(x, y);
-            cv::circle(baseimage, centr_all, radius, black,1, CV_AA);
-        }
-    };
-
-
-
-    virtual void resetImage(cv::Mat &image){
-        baseimage.copyTo(image);
-    };
```

Comment:
Is this part of dead code for skin visualization being removed? Couldn't find replacements being added in any other files.